Bottom: 1116961644fe14b778a705068cc3ad1dac5c3eca
Top:    c7118b24e57cf9dadbc32d3e46fb56a74907a0d8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-03 15:57:04 +0100

Refresh of hid-gtk-don-t-use-a-gtkeventbo

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index f327573..fb5a32a 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -1308,7 +1308,8 @@ ghid_build_pcb_top_window (void)
    * a modal button GetLocation button press.
    */
   ghidgui->left_toolbar = gtk_vbox_new (FALSE, 0);
-  gtk_box_pack_start (GTK_BOX (hbox_middle), ghidgui->left_toolbar, FALSE, FALSE, 3);
+  gtk_box_pack_start (GTK_BOX (hbox_middle),
+                      ghidgui->left_toolbar, FALSE, FALSE, 3);
 
   vbox = ghid_scrolled_vbox (ghidgui->left_toolbar, &scrolled,
                              GTK_POLICY_NEVER, GTK_POLICY_AUTOMATIC);
